specifyweb/frontend/js_src/lib/components/Statistics/index.tsx

```diff
@@ -0,0 +1,140 @@
+    React.useCallback(
+      async () =>
+        ajax<BackendStatsResult>('/statistics/collection/global/', {
+          method: 'GET',
+          headers: {
+            Accept: 'application/json',
+          },
+        }).then(({ data }) => data),
+      []
+    ),
+    false
+  );
+  return backendStatObject;
+}
+
+export function StatsPage(): JSX.Element {
+  const statsObjectCombined = statsSpec;
+  const backEndResult = useBackendApi();
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        height: '100%',
+        gap: '1rem',
+        flexDirection: 'column',
+        margin: '0 auto',
+        padding: '5%',
+        paddingTop: '0',
+      }}
+      className="bg-gray-200"
+    >
+      <h1
+        style={{
+          fontSize: '2rem',
+          fontWeight: 'initial',
+        }}
+      >
+        Collection Statistics
+      </h1>
+
+      <div
+        style={{
+          display: 'grid',
+          gridTemplateColumns: 'repeat(auto-fill, minmax(20rem,1fr))',
+          gridTemplateRows: 'masonry',
+          gap: '1rem',
+        }}
+      >
+        {Object.entries(statsObjectCombined).map(function getStatNameCat([
+          categoryName,
+          { label, categories },
+        ]) {
+          const statObject = (
+            categories as (
+              backendStatsResult:
+                | BackendStatsResult[typeof categoryName]
+                | undefined
+            ) => StatCategoryReturn
+          )(backEndResult?.[categoryName]);
+          return (
+            <div
+              className="border-[1px] bg-white"
+              style={{
+                display: 'block',
+                borderColor: 'black',
+                borderRadius: '1rem',
+                padding: '1rem',
+                alignContent: 'center',
+              }}
+            >
+              <h1>
+                <b>{label}</b>
+              </h1>
+
+              {statObject === undefined
+                ? commonText('loading')
+                : Object.entries(statObject).map(function getstatValue([
+                    key,
+                    { label, spec },
+                  ]) {
+                    const statValue =
+                      spec.type === 'Querybuildstat' ? (
+                        <QueryBuilderStat
+                          tableName={spec.tableName}
+                          fields={spec.fields}
+                        />
+                      ) : (
+                        spec.value ?? commonText('loading')
+                      );
+                    return (
+                      <div
+                        key={key}
+                        style={{
+                          display: 'flex',
+                          flexDirection: 'row',
+                          justifyContent: 'space-between',
+                        }}
+                      >
+                        <h4>{label}</h4>
+                        <h6>{statValue}</h6>
```

Comment:
Bad. Bad. Bad.

First, don't skip levels. If you have `h4`, then you should have `h5` after it, not `h6`.
Secondly, stat value is not a head, and I would argue stat label is not a header either.

Instead, consider replacing the parent `<div>` with a `<p>`, and replacing both `<h4>` and `<h6>` with a `<span>` (span does not have semantic meaning, but allows `justifyContent: 'space-between',` to work like expected)